### PR TITLE
fix: removing cache to accelerate testing cycle

### PR DESCRIPTION
**Pull request by bradegler**

### CODE REVIEW

**Review by verbanicm — APPROVED** [review]

**Review by verbanicm — APPROVED** [review]

File: pkg/config/config_evaluator.go [hunk 1]
```diff
@@ -46,24 +46,43 @@ func NewConfigEvaluator(expireAt time.Duration, localConfigDir, repoConfigPath,
 	if err != nil {
 		return nil, fmt.Errorf("failed to create CEL environment: %w", err)
 	}
+	enableCaching := false
```
Comment (sethvargo) [inline pkg/config/config_evaluator.go:49]: Drive-by. I would prefer if we had a runtime config for "cache duration" and we could set it to something super small like "1s"

Comment (sethvargo) [reply pkg/config/config_evaluator.go:49]: @bradegler ^

Comment (bradegler) [reply pkg/config/config_evaluator.go:49]: Yes. It will get moved into a command line option. I have that already teed up in a follow up.
